packages/@storefront/core/package.json

```diff
@@ -4,12 +4,12 @@
   "description": "Core library for StoreFront components",
   "repository": {
     "type": "git",
-    "url": "https://github.com/groupby/storefront-core.git"
+    "url": "https://github.com/groupby/storefront.git"
   },
   "bugs": {
-    "url": "https://github.com/groupby/storefront-core/issues"
+    "url": "https://github.com/groupby/storefront/issues"
   },
-  "homepage": "https://github.com/groupby/storefront-core#readme",
+  "homepage": "https://github.com/groupby/storefront/tree/master/packages/%40storefront/core",
   "author": "Ben Teichman <ben.teichman@groupbyinc.com>",
```

Comment:
Should we change the author to a generic user? If so we need to add an author to every package.json. Otherwise in order to keep it consistent maybe we should just remove authors?

Reply:
I've spun up the following ticket to track author-related updates: https://groupby.atlassian.net/browse/SF-1242.

Marking this conversation as resolved.